### PR TITLE
Added dip 16 wide footprinter3d example and fixed length of dip model

### PR DESCRIPTION
**Pull request by anas-sarkez**

### CODE REVIEW

File: lib/DualInlinePackage.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export const Dip = ({
       })}
       <ChipBody
         width={bodyWidth}
-        length={(10 / 4) * numPinsOnEachSide}
+        length={numPinsOnEachSide * pitch + 0.5}
```
Comment (seveibar) [inline lib/DualInlinePackage.tsx:99]: on a plane but are you sure it’s not this? Would make more sense than adding 0.5mm

```suggestion
        length={(numPinsOnEachSide + 0.5) * pitch}
```

**Review by seveibar — APPROVED** [review]

File: lib/DualInlinePackage.tsx (hunk 1 above)
Comment (anas-sarkez) [reply lib/DualInlinePackage.tsx:99]: Screenshot 2024-10-11 at 10 24 13 PM
According to this ref

Comment (anas-sarkez) [reply lib/DualInlinePackage.tsx:99]: Screenshot 2024-10-11 at 10 28 42 PM
@seveibar 
oops! I think we have many types

Comment (seveibar) [reply lib/DualInlinePackage.tsx:99]: Makes sense

**Review by seveibar — APPROVED** [review]

**Pull request merged**